docs/source/upcoming_release_notes/1255-add_preset_support_for_SQR1.rst

```diff
@@ -0,0 +1,30 @@
+IssueNumber Title
```

Comment:
did you run the script to generate this file?  I'm surprised the filename is standard but the title here isn't